docker-compose.nobuild.yml

```diff
@@ -12,6 +13,16 @@ services:
     - MICRONAUT_SERVER_CORS_ENABLED='true'
     ports:
     - '8082:8082'
+  app-client:
+    image: louisga/cdc-client:1.0
+    container_name: cdc-client
+    ports:
+    - '8000:8081'
+    depends_on:
+    - app-service
+    environment:
+    - USERS_URL=http://localhost:8082/api/users
+    - EVENTS_STREAMING_URL=http://localhost:8082/api/users/stream
```

Comment:
The URLs here will not be resolved in the containers. The hosts should be changed to `app-service`

Reply:
Resolved for _USERS_URL_
_EVENTS_STREAMING_URL_ will still have to be **localhost**